Lib/http/cookies.py

```diff
@@ -281,6 +281,7 @@ class Morsel(dict):
         "secure"   : "Secure",
         "httponly" : "HttpOnly",
         "version"  : "Version",
+        "samesite" : "SameSite"
```

Comment:
Include a trailing comma so that if more items are added later, this line doesn't have to be modified again.